ocn_027b_rw0_wastewater_pourpoints/ocn_027b_rw0_wastewater_pourpoints_processing.py

```diff
@@ -0,0 +1,112 @@
+import geopandas as gpd
+import os
+import pyproj
+from shapely.geometry import Point
+import urllib.request
+import sys
+
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+
+from cartoframes.auth import set_default_credentials
+from cartoframes import to_carto, update_privacy_table
+import util_files
+import util_cloud
+from zipfile import ZipFile
+import logging
+
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: 
+    logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of table on Carto where you want to upload data
+# this should be a table name that is not currently in use
+dataset_name = 'ocn_027b_rw0_wastewater_pourpoints'
+
+logger.info('Executing script for dataset: ' + dataset_name)
+# create a new sub-directory within your specified dir called 'data'
+# within this directory, create files to store raw and processed data
+data_dir = util_files.prep_dirs(dataset_name)
+
+'''
+Download data and save to your data directory
+'''
+logger.info('Downloading raw data')
+# insert the url used to download the data from the source website
+url = 'https://knb.ecoinformatics.org/knb/d1/mn/v2/object/urn%3Auuid%3Aaf8d0bd6-dc0c-4149-a3cd-93b5aed71f7c'
+
+# download the data from the source
+raw_data_file = os.path.join(data_dir, 'N_PourPoint_And_Watershed.zip')
+urllib.request.urlretrieve(url, raw_data_file)
+
+# unzip source data
+raw_data_file_unzipped = raw_data_file.split('.')[0]
+zip_ref = ZipFile(raw_data_file, 'r')
+zip_ref.extractall(raw_data_file_unzipped)
+zip_ref.close()
+
+
+'''
+Process data
+'''
+
+# load in the polygon shapefile
+shapes = os.path.join(raw_data_file_unzipped, 'effluent_N_pourpoints_all.shp')
```

Comment:
Can you use `processed_data_file` here instead of `shapes` for consistency with our other scripts?

Reply:
Actually I think the main thing is that the final thing you upload is called `processed_data_file`.

Reply:
Ive updated this section, let me know if it looks ok